_targets.R

```diff
@@ -89,11 +89,12 @@ GFv1_HRUs_edited_url <- "https://github.com/USGS-R/drb-network-prep/blob/940073e
 # Define USGS stat codes for continuous sites that only report daily statistics (https://help.waterdata.usgs.gov/stat_code) 
 stat_cd_select <- c("00001","00003")
 
-# Define earliest startDate for NWIS data retrievals
+# Define earliest startDate and latest endDate for NWIS data retrievals
 earliest_date <- "1979-10-01"
+latest_date <- "2021-12-31"
 
 # Change dummy date to force re-build of NWIS SC sites and data download
-dummy_date <- "2021-12-31"
+dummy_date <- "2022-06-16"
```

Comment:
Given the addition of `NWIS_cue` in this PR, do you expect that `dummy_date` ever gets used in the pipeline? Should this variable be omitted, or is it still useful as a sort of documentation?

Reply:
If we remember to change this whenever we change the NWIS_cue, then it does serve as documentation of the date we last updated the NWIS data. I'm going to move the dummy_date to the line below NWIS_cue so that it encourages us to remember.

Reply:
Good idea, and is there a reason `NWIS_cue` is defined in 1_fetch.R rather than with the other variables in _targets.R?

Reply:
The pipeline did not build when I had NWIS_cue in _targets.R. I'm not sure why that resulted in a variable not found error. It works when it's in 1_fetch.R